e2e/cypress/integration/account_settings/sidebar/channel_switcher_spec.js

```diff
@@ -211,4 +211,48 @@ describe('Account Settings > Sidebar > Channel Switcher', () => {
         // * Channel name should be visible in LHS
         cy.get(`#sidebarItem_${testChannel.name}`).should('be.visible');
     });
+
+    it('Cmd/Ctrl+Shift+L closes Channel Switch modal and sets focus to post textbox', () => {
+        // # Go to a known team and channel
+        cy.visit('/ad-1/channels/town-square');
+        cy.get('#channelHeaderTitle').should('be.visible').should('contain', 'Town Square');
+
+        // # Type CTRL/CMD+K
+        cy.get('#post_textbox').cmdOrCtrlShortcut('K');
+
+        // * Channel switcher hint should be visible
+        cy.get('#quickSwitchHint').should('be.visible').should('contain', 'Type to find a channel. Use ↑↓ to browse, ↵ to select, ESC to dismiss.');
+
+        // # Type CTRL/CMD+shift+L
+        cy.get('#quickSwitchInput').cmdOrCtrlShortcut('{shift}L');
+
+        // * Suggestion list should be visible
+        cy.get('#suggestionList').should('not.be.visible');
+
+        // * focus should be on the input box
+        cy.get('#post_textbox').should('be.focused');
+        cy.get('#post_textbox').should('be.empty');
```

Comment:
Just curious if this passed on you.  On my machine (Mac), it's failing due to `L` being typed into the `#post_textbox`.

Reply:
no, it is passing on my end, some of the other tests in the spec fail occasionally, but not these

Reply:
I rechecked the snapshots just to make sure, and neither test add any input to the textbox

Reply:
I ran this with `it.only` and it's consistently failing but will just keep an eye on it (will let you know the result during Cypress daily test)